database-couchdb/index.js

```diff
@@ -0,0 +1,103 @@
+const fs = require('fs');
+const PATH = require('path');
+const csvParser = require('csv-parser');
+const Axios = require('axios');
+
+const availabilityPath = PATH.resolve(__dirname, '..', 'data-gen', 'csv', 'availability.csv');
+const storesPath = PATH.resolve(__dirname, '..', 'data-gen', 'csv', 'stores.csv');
+
+const startingItemId = 100;
+const batchSize = 100;
```

Comment:
batchSize means ur uploading documents to couch db 100 at a time? I read you could bulk insert 100,000 at a time? Why'd you settle on 100? How long did seeding take? More for my own curiosity

Reply:
LOL this was through hella trial and error. I was overloading my laptop when my batch uploads were more than 100 at a time (even with my set timeouts being 10000 ms). For the final parameters i went with, it only took about 20-30 min i think.

I was getting a lot of 400 or 500 status codes before that. Also it took my laptop a long time to optimize/ compress the database after it was seeded, but I think thats just a couchdb thing. It was using 400% of my cpu for a few hours...